.storybook/package.json

```diff
@@ -12,7 +12,6 @@
   "dependencies": {
     "@adobe/spectrum-css-workflow-icons": "^1.5.4",
     "@spectrum-css/expressvars": "^3.0.9",
-    "@spectrum-css/quickaction": "^3.1.1",
```

Comment:
@mdt2 I confirmed that the Card and Quick action stories don't break when we remove this from dependencies and they looked good. The root requirement is sufficient for pulling the old styles in for storybook and the docs site.

Reply:
Perfect, thanks for cleaning this up!